src/HealthChecks/HealthChecks/src/PublicAPI.Shipped.txt

```diff
@@ -22,20 +22,20 @@ Microsoft.Extensions.Diagnostics.HealthChecks.HealthCheckService.HealthCheckServ
 Microsoft.Extensions.Diagnostics.HealthChecks.HealthCheckServiceOptions
 Microsoft.Extensions.Diagnostics.HealthChecks.HealthCheckServiceOptions.HealthCheckServiceOptions() -> void
 Microsoft.Extensions.Diagnostics.HealthChecks.HealthCheckServiceOptions.Registrations.get -> System.Collections.Generic.ICollection<Microsoft.Extensions.Diagnostics.HealthChecks.HealthCheckRegistration!>!
-static Microsoft.Extensions.DependencyInjection.HealthChecksBuilderAddCheckExtensions.AddCheck(this Microsoft.Extensions.DependencyInjection.IHealthChecksBuilder! builder, string! name, Microsoft.Extensions.Diagnostics.HealthChecks.IHealthCheck! instance, Microsoft.Extensions.Diagnostics.HealthChecks.HealthStatus? failureStatus = null, System.Collections.Generic.IEnumerable<string!>? tags = null, System.TimeSpan? timeout = null) -> Microsoft.Extensions.DependencyInjection.IHealthChecksBuilder!
+static Microsoft.Extensions.DependencyInjection.HealthChecksBuilderAddCheckExtensions.AddCheck(this Microsoft.Extensions.DependencyInjection.IHealthChecksBuilder! builder, string! name, Microsoft.Extensions.Diagnostics.HealthChecks.IHealthCheck! instance, Microsoft.Extensions.Diagnostics.HealthChecks.HealthStatus? failureStatus = null, System.Collections.Generic.IEnumerable<string!>? tags = null, System.TimeSpan? timeout = null, System.TimeSpan? period = null) -> Microsoft.Extensions.DependencyInjection.IHealthChecksBuilder!
```

Comment:
Nothing should have changed in this Shipped file, that would indicate an API breaking change. Even adding a new optional parameter qualifies as an API break. This PR should only be adding new APIs, and they should show up in the Unshipped file.